CocoIDE/cocol.py

```diff
@@ -160,22 +177,22 @@ def linkFiles(self, event=None, sym=False):
                 tk.END,
                 "\n\nLINKED OK! Image written to:\n " + objfiles[0][:-4] + ".img\n",
             )
-            self.statusText.insert(tk.END, "\nLINKER REPORT LISTING:\n" + listing)
+            self.statusText.insert(
+                tk.END, "\nLINKER REPORT LISTING:\n" + listing
+            )  # noqa
             self.statusText.see(tk.END)
         # while int(float(self.statusText.index("end linestart ")))>11: # Scroll
         #    #print("link files", int(float(self.statusText.index("end linestart "))))
         #    self.statusText.delete("1.0", "1.0 lineend +1c")# Scroll
 
-    def closeCocol(self):
+    def closeCocol(self) -> None:
         self.mainWin.destroy()
         if __name__ == "__main__":
             self.master.destroy()
 
 
 # Functions
-
-
-def EP(s):
+def EP(s) -> None:  # The type of s is ambiguous here
```

Comment:
Is it? It's `str`, you know this for two reasons:
1. How this (questionable) function is used
2. The next few lines don't TypeError